lib/PreviewProvider.js

```diff
@@ -36,6 +38,7 @@ const DEFAULT_OPTIONS = {
 function PreviewProvider(options = {}) {
   this.options = Object.assign({}, DEFAULT_OPTIONS, options);
   this._onPrefChange = this._onPrefChange.bind(this);
+  this._tippyTopProvider = new TippyTopProvider();
```

Comment:
hmm. yeah. I think in the long run we want to refactor this code to something more flexible to create a sort of pipeline to process the sites data. But are you suggesting I change something here?